src/components/ha-alert.ts

```diff
@@ -83,7 +83,6 @@ class HaAlert extends LitElement {
       position: relative;
       padding: 8px;
       display: flex;
-      margin: 4px 0;
```

Comment:
How is this affecting other places?

Reply:
I need to check that. We should never have a margin on the outside of a component. It's up to the parent how different components need to be styled together.

Reply:
Fixed the design docs CSS. It was rendering multiple next to one another. Shouldn't happen in real code.